### ISSUE TITLE
Graphics Issue

### ISSUE THREAD

**Issue by Gamerbolts** [Funwayguy/StandardQuestingPack#76]
Hi, I updated my modpack from 1.7.10 to 1.12.2 finally and brought over the questline, when I was looking through the questbook I found this https://imgur.com/a/zgptsBb

Not sure if its a common problem I do know when I load the game I get:

13:33:41 | betterquesting | main | fatal | [!] FAILED TO ENABLE STENCIL BUFFER. GUIS WILL BREAK! [!]
That is the only critical error in the log.

I am using the most update versions of BQ and Standard Expansion

Any help is appreciated, Thank you

**Comment by Gamerbolts** [comment]
Any help on this? Newest version still does this to my GUI the text is just overlapping

**Comment by Funwayguy** [comment]
What GPU are you running this through? The inability to use the stencil buffer is usually hardware related. Previous versions of BQ used to use a different method that didn't require newer hardware but it posed a number of limitations and issues.

**Comment by Gamerbolts** [comment]
I have a Nvidia GTX 1050ti I was checking about stencil buffers but couldn't find any information

**Comment by Funwayguy** [comment]
Odd, I've tested with an GTX1060 and a RX 560 and both seem to support it just fine. Are you absolutely sure your PC is set to run Minecraft on the GPU and not internal graphics? 

F3 might help in showing what Minecraft is running on.

**Comment by Gamerbolts** [comment]
I am running dual graphics cards, the 1050ti and a 550ti. with two monitors. I know for a fact though that the 1050ti takes precedence for Minecraft.

**Comment by Gamerbolts** [comment]
If you would like I could send you a link to my modpack, its private on the Technic Launcher, You can see if it is doing the same things to you?

**Comment by Funwayguy** [comment]
Might be worth a shot but I can't test it right now (1am where I am). When I get time I'll throw it on my GTX1060 PC and see what happens

**Comment by Gamerbolts** [comment]
No problem, not in a hurry, pack is far from release lol

http://api.technicpack.net/modpack/bolts-crazzzy-adventure

Thank you for the help!

**Comment by Funwayguy** [comment]
Appears I'm not authorised to download the pack (assuming because it's private) is there somewhere else you'd be willing to DM me a test version?

**Comment by Gamerbolts** [comment]
oh with Technic you have to copy the link and open the launcher then paste it in the search bar.

**Comment by Funwayguy** [comment]
Off to an interesting start...

![image](https://user-images.githubusercontent.com/6048596/59831232-b7cad400-9384-11e9-9dcf-1c4384357db9.png)

Moved it to MultiMC because I couldn't change the version of Forge it launched with and...

![image](https://user-images.githubusercontent.com/6048596/59833239-024e4f80-9389-11e9-8404-592b01619fc0.png)

This was on my RX 560X dev laptop. Not sure if running it through MultiMC made a difference unless Technic is tampering with something it shouldn't (wouldn't surprise me though).

PS: These pack controls are wack. Had to look at the controls menu to figure it out.

**Comment by Gamerbolts** [comment]
Ugh... that is why this pack is in alpha state lol. Sorry about that.

Hmm that is interesting, I know with testing it I was doing it on the twitch/curse launcher. I will try it on MultiMC and see if that has any difference for me.

I think i accidentally had my controls saved, 'F' for sneak 'SHIFT' for jump? That is how I have played Minecraft for years lol

**Comment by Gamerbolts** [comment]
This is what I am getting on mine
![image](https://user-images.githubusercontent.com/20451601/59859583-9cd77000-9342-11e9-844b-89c1bb1e2960.png)

So it must be my graphics card... hmm. Thank you for the help

"edit" i noticed the rewards were different, I didn't update the pack... whoops i'll get a pack update out maybe it was something with the update... not sure

**Comment by Funwayguy** [comment]
I feel like I would have gotten considerable amounts of complaints if it was strictly a GPU issue.

That said if OpenGL is silently seeing the 550ti's older limitations and enforcing them across both cards (for SLI reasons?) even if you're only using one, maybe there could be something there.

**Comment by Gamerbolts** [comment]
Yeah that could definitely be it. My friend will be giving me a 970 GTX FTW that i'll dual with the 1050ti. and can finally get rid of the 550ti. When I do that i'll test again, come back on here and let you know.

Thanks again for all of your help.

**Comment by Funwayguy** [comment]
Just a quick update on this. Had another user report the same thing and it turns out it was Optifine in some way. No idea why or if it was a particular setting, or why it worked fine on my PC, but that might fix it for you if you're also using Optifine.

EDIT: Specific option is "Fast Render"

**Comment by Gamerbolts** [comment]
Should of guessed Optifine was responsible in some way, 

I took it out and it works. I put it back in and turned off "Fast Render" that did not work. So for whatever reason Optifine is messing with Better Questing.
